lib/requester/browser/request.js

```diff
@@ -12,6 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+/* eslint-disable */
```

Comment:
?

Reply:
It wouldn't be fixable without having to completely touch every line of the file. I'm not sure if we have such test coverage for the browser request flows.

Reply:
Fine, to be done in a separate P.R then.